app/src/main/res/layout/layout_component_switch.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.SwitchCompat xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should we use `com.google.android.material.switchmaterial.SwitchMaterial` here? 🙂

Reply:
For sure